flujo/builtins_optional.py

```
@@ -122,7 +122,11 @@ async def render_jinja_template(template: str, variables: Optional[JSONObject] =
         if _jinja2 is None:
             return template
         try:
-            env = _jinja2.Environment(undefined=_jinja2.StrictUndefined, autoescape=False)
+            try:
+                from jinja2.sandbox import SandboxedEnvironment
+            except Exception:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)